src/pe_reports/data/db_query.py

```diff
@@ -10,13 +11,10 @@
 from psycopg2 import OperationalError
 from psycopg2.extensions import AsIs
 
-# cisagov Libraries
-from pe_reports import app
-
 from .config import config
 
 # Setup logging to central file
-LOGGER = app.config["LOGGER"]
+LOGGER = logging.getLogger(__name__)
```

Comment:
We are calling the Logger this way because importing app has a circular import error. The logs still go to the central logging file, this just avoids the circular import.

Reply:
Got it, thanks for the explanation.  Can you please add a comment here so that future readers of this code understand why you are doing it this way?

Reply:
@dav3r comment added in the following commit: [9ea3769](https://github.com/cisagov/pe-reports/pull/282/commits/9ea37695b812308973b8dc71a37b72c18ea8fc47)